config/200-bootstrap.yaml

```diff
@@ -93,7 +93,7 @@ data:
               explicit_http_config:
                 http2_protocol_options:
                   connection_keepalive:
-                    interval: 30s
+                    interval: 300s
```

Comment:
Hmm... I thought that we don't need to increase the interval as we set `PermitWithoutStream` to true. Do we still need this change?

Reply:
`PermitWithoutStream` is a setting that allows ping when there is no active stream.
So it is necessary to extend interval to `EnforcementPolicy.MinTime`.